packages/scaffold-config/src/dependencies.ts

```diff
@@ -97,6 +97,51 @@ export const WIZARD_DEPENDENCY_NEXT = {
   minVersion: '>=10.0.0',
 } as const
 
+export const WIZARD_DEPENDENCY_ANGULAR_CLI = {
+  type: 'angular',
+  name: 'Angular CLI',
+  package: '@angular/cli',
+  installer: '@angular/cli',
+  description: 'CLI tool that you use to initialize, develop, scaffold, and maintain Angular applications.',
+  minVersion: '>=13.0.0',
```

Comment:
Do we also want to include a `maxVersion`? (not sure we actually have code path supporting this, but it would be easy to add, I think).

Also, do we know this does **not** work with any other versions (like 12 or 14)?

Reply:
Actually we can add a `maxVersion` here: https://github.com/cypress-io/cypress/issues/22890

Question still stands about other versions - do we have support for 12 or  14?

Reply:
No support for 12 but 14 works (and is proven out in the system tests)